sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageArrowReader.java

```diff
@@ -43,9 +42,7 @@
   BigQueryStorageArrowReader(ReadSession readSession) throws IOException {
     protoSchema = readSession.getArrowSchema();
     InputStream input = protoSchema.getSerializedSchema().newInput();
-    this.arrowBeamSchema =
-        ArrowConversion.ArrowSchemaTranslator.toBeamSchema(
-            ArrowConversion.arrowSchemaFromInput(input));
+    ArrowConversion.ArrowSchemaTranslator.toBeamSchema(ArrowConversion.arrowSchemaFromInput(input));
```

Comment:
I'm trying to understand what's the purpose of this line. Maybe it verifies that the schema can be converted to a Beam schema?

do you know? @vachan-shetty  @kmjung

Reply:
The read API team didn't add the Arrow code here, so I'm not totally sure. Looking...

Reply:
I worked with @MiguelAnzoWizeline on this. I think it really is dead code, the other BigQueryStorageReader implementation doesn't verify it can be converted to a Beam schema here: https://github.com/apache/beam/blob/9160ba2dfb34ce4cdf1760c24b6e57fb912eb5c1/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryStorageAvroReader.java#L40-L46

Reply:
I removed the line